modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -34,14 +34,16 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
-  profiles:
-  - AffinityAndTaints       <1>
-  - TopologyAndDuplicates   <2>
-  - LifecycleAndUtilization <3>
+  profileCustomizations:
+    podLifetime: 30m        <1>
+  profiles:                 <2>
+  - AffinityAndTaints
+  - TopologyAndDuplicates   <3>
+  - LifecycleAndUtilization
 ----
-<1> Enable the `AffinityAndTaints` profile, which evicts pods that violate inter-pod anti-affinity, node affinity, and node taints.
-<2> Enable the `TopologyAndDuplicates` profile, which evicts pods in an effort to evenly spread similar pods, or pods of the same topology domain, among nodes.
-<3> Enable the `LifecycleAndUtilization` profile, which evicts long-running pods and balances resource usage between nodes.
+<1> Optional: Enable a custom pod lifetime value for the `LifecycleAndUtilization` profile. The default pod lifetime is 24 hours.
```

Comment:
Might be worth it to call out [`Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"`](https://pkg.go.dev/time#ParseDuration)